libbeat/logp/configure/logging.go

```diff
@@ -30,18 +31,22 @@ var (
 	verbose        bool
 	toStderr       bool
 	debugSelectors []string
+	environment    logp.Environment
 )
 
+type environmentVar logp.Environment
+
 func init() {
 	flag.BoolVar(&verbose, "v", false, "Log at INFO level")
 	flag.BoolVar(&toStderr, "e", false, "Log to stderr and disable syslog/file output")
 	common.StringArrVarFlag(nil, &debugSelectors, "d", "Enable certain debug selectors")
+	flag.Var((*environmentVar)(&environment), "environment", "set environment the Beat is run in")
```

Comment:
We should add docs about this new flag.

Reply:
It is kind of a private flag (a hack) I just introduced as a workaround. I rather wish to remove the flag again in the future. Maybe for 8.0 we have to rethink some logging defaults. e.g. maybe we can detect the environment we are running in, or just log to stdout/stderr by default.

Reply:
Yep, I think we could consider logging to stdout/stderr by default on 8.0.